src/api/index.ts

```diff
@@ -13,9 +15,12 @@ export const requestCategories = () =>
 
 export const requestNotes = () =>
   new Promise((resolve, reject) => {
-    const data = localStorage.getItem('notes') || '[]'
+    const data = localStorage.getItem('notes')
 
-    if (data) {
+    if (!data) {
```

Comment:
If they delete the note and have no data, will the welcomeNote pop in every time?

Reply:
No, the welcomeNote shouldn't pop up since there is an empty array in the localStorage if the user removes all notes, but if the user never visited the site before, i.e. localStorage is empty, it will display the welcomeNote.